frontend_tests/node_tests/settings_muted_users.js

```diff
@@ -22,7 +22,7 @@ run_test("settings", ({override}) => {
     override(list_widget, "create", ($container, list) => {
         assert.deepEqual(list, [
             {
-                date_muted_str: "Jan\u00A001,\u00A02020",
+                date_muted_str: "Jan 01, 2020",
```

Comment:
Do you understand why the `\u00A0` characters were there in this previously? And does the new version have one of those, or a normal space? 

https://www.fileformat.info/info/unicode/char/00a0/index.htm

Reply:
I understand what this character does. I guess it's here to prevent line breaking on small mobile screens. 

All this situation with NBSP and special characters is a bit tricky. It may be risky to remove them outright. But also, I think this is not the right place for them. If we don't want lines to break, we could control it via the parent container's CSS property `white-space: nowrap`. Also, I double-checked the dates and they look good. Sometimes even better, as seen in the screenshots below:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/53193850/211556599-b809d2b7-3d40-4521-a876-1d6853bfad86.png">
<img width="303" alt="image" src="https://user-images.githubusercontent.com/53193850/211556738-0f96bfa6-cc29-46b5-97fa-2283df06d15a.png">

We should decide whether to remove these characters or leave them, as it affects implementation. In my opinion, it would be better to remove them from here.

Reply:
Yeah, those leading 0s on days are almost certainly a mistake.

Reply:
Ahh, good point, we should be able to just do non-breaking spaces via CSS, since we have just the date in a cell, and the previous logic used a single space character everywhere. OK, sounds good.

@alya FYI.